### PR TITLE
[14.0][MIG] sale_report_delivered_brand: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0

Locked by:
- [x] `sale_report_delivered`: https://github.com/OCA/sale-reporting/pull/185

Please @CarlosRoca13 and @chienandalu can you review it?

@Tecnativa TT40102

### CODE REVIEW

File: sale_report_delivered_brand/__manifest__.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
     "name": "Sale Report Delivered Brand",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-reporting",
     "category": "Sales",
     "license": "AGPL-3",
-    "depends": ["sale_report_delivered", "product_brand"],
-    "data": ["views/sale_report_delivered_views.xml"],
+    "depends": ["sale_report_delivered_subtotal", "product_brand"],
```
Comment (CarlosRoca13) [inline sale_report_delivered_brand/__manifest__.py]: Why depending of `sale_report_delivered_subtotal`?

File: sale_report_delivered_brand/reports/sale_report.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    product_brand_id = fields.Many2one(comodel_name="product.brand", string="Brand")
```
Comment (chienandalu) [inline sale_report_delivered_brand/reports/sale_report.py]: ¿Don't we need to override the sql view?

**Comment by victoralmau** [comment]
Changes done, could you review it again?

File: sale_report_delivered_brand/README.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+    * Carlos Roca
```
Comment (CarlosRoca13) [inline sale_report_delivered_brand/README.rst:64]: Add yourself as contributor

**Review by CarlosRoca13 — APPROVED** [review]
Just an optional comment

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**